src/main/java/org/sagebionetworks/web/server/servlet/UserAccountServiceImpl.java

```diff
@@ -54,18 +54,6 @@ private void validateService() {
     }
   }
 
-  @Override
-  public void signTermsOfUse(String accessToken) throws RestServiceException {
-    validateService();
-
-    SynapseClient synapseClient = createSynapseClient();
-    try {
-      synapseClient.signTermsOfUse(accessToken);
```

Comment:
SWC-7152: New Synapse Client does not have a method with this signature (you must now provide the ToS version as well!).  We don't use a GWT servlet to handle this call now (since it's in the Sage Account web app), so we can delete this dead code.